src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -348,7 +348,7 @@ def _search(self, ldap_config, basedn='', scope=pyldap.SCOPE_SUBTREE, filter='',
                     attrsonly=0,
                     serverctrls=serverctrls,
                     clientctrls=clientctrls,
-                    timeout=timeout,
+                    timeout=ldap_config['options']['timeout'],
```

Comment:
Actually, wondering if we should make sure this is some sane value (instead of like 9999999).